cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -463,13 +471,25 @@ func (p *BicepProvider) Destroy(
 					return p.purgeAPIManagement(ctx, apiManagements, options, skipPurge)
 				},
 			}
+
 			var purgeItem []itemToPurge
 			for _, item := range []itemToPurge{keyVaultsPurge, appConfigsPurge, aPIManagement} {
 				if item.count > 0 {
 					purgeItem = append(purgeItem, item)
 				}
 			}
 
+			// cognitive services are grouped by resource group because the name of the resource group is required to purge
+			for resourceGroup, cogAccounts := range cognitiveAccounts {
+				purgeItem = append(purgeItem, itemToPurge{
+					resourceType: "Cognitive account from resource group: " + resourceGroup,
```

Comment:
This resource type naming is inconsistent with other resources. Should we fix all, or should we follow the current pattern?

Reply:
Updated to use the same approach than the others.
It looks like
![image](https://github.com/Azure/azure-dev/assets/24213737/2332f523-83a0-43be-92df-2d42d1a2b189)